docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -0,0 +1,96 @@
+---
+assignees:
+- mikedanese
+
+---
+
+* TOC
+{:toc}
+
+## Summary
+
+This document describes setting up TLS client certificate bootstrapping for kubelets.
```

Comment:
"This document describes how to set up the TLS client certificate boostrapping for kubelets."